web/template/layout/pro-deputy.gotmpl

```diff
@@ -1,4 +1,5 @@
 {{ define "pro-deputy" }}
+    <a href="javascript: history.go(-1)" class="govuk-back-link">Back</a>
```

Comment:
Same here. We should be using urls for navigation, not browser history

Reply:
The back button is in the prototype though, I can't see a way to code it in other than this? It's in the deputy details sub template which spans like 8 or 9 templates so it would be a lot of code to hard code it as an if statement with all those other options.

Reply:
Doesn't that back button only ever take you to the dashboard though? My understanding was that its a "back to the dashboard button", not a "back to the previous page" one. It would be worth clarifying with Jen, but I don't think we should be replicating browser behaviour in our UI. That leads to confusion for users.

Reply:
Ah my bad, because it just went back on the prototype I assumed that was the design. I'll double check.